cs-vmware/src/main/java/io/cloudslang/content/vmware/utils/InputUtils.java

```diff
@@ -138,4 +139,10 @@ public static void checkMutuallyExclusiveInputs(final String input1, final Strin
             throw new IllegalArgumentException(exceptionMessage);
         }
     }
+
+    public static void checkOptionalMutuallyExclusiveInputs(final String input1, final String input2, final String exceptionMessage) {
+        if (isNotBlank(input1) && isNotBlank(input2)) {
```

Comment:
does this check mutually exclusivity? the method name is confusing 
if yes, this doesn't check if both are empty